.circleci/config.yml

```diff
@@ -107,47 +83,87 @@ jobs:
           path: /tmp/artifacts/
   build-docker-image:
     docker:
-      - image: circleci/node:8
+      - image: circleci/buildpack-deps:xenial
     steps:
       - setup_remote_docker
       - checkout
-      - run: sudo apt install python-pip
       - run: sudo pip install -r requirements_bundles.txt
-      - run: .circleci/update_version
+      - run: .circleci/update_version "master"
       - run: npm run bundle
-      - run: .circleci/docker_build
+      - run: .circleci/docker_build "master"
+  build-docker-image-rc:
```

Comment:
Those `build-docker-image*` jobs would need to follow the same pattern as the `build-docker-image` job above.